src/test/dbwrapper_tests.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2012-2013 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "dbwrapper.h"
+#include "uint256.h"
+#include "random.h"
+#include "test/test_pivx.h"
+
+#include <boost/assign/std/vector.hpp> // for 'operator+=()'
+#include <boost/assert.hpp>
+#include <boost/test/unit_test.hpp>
+
+using namespace std;
+using namespace boost::assign; // bring 'operator+=()' into scope
+using namespace boost::filesystem;
```

Comment:
Can we update this to follow our policy of not introducing things into the global namespace? (ref: https://github.com/bitcoin/bitcoin/commit/4cfd57d2e38207d78722ce8c9274ba8dd700d1cc#diff-ac09ab7f710fc2fb7850f59601545636).

Otherwise its looking good

Reply:
Sure. Done.